filter-assets.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var _ = require('lodash');
+
+var filterTypes = {
+  js: 'js',
+  css: 'css',
+  fonts: 'ttf', // ??
+  images: ['png', 'jpeg', 'jpg', 'gif']
```

Comment:
Pretty sure this is non exhaustive…but I guess the main point is to filter these _out_ in favour of only getting CSS or JS files. Probably still worth getting it complete though…is that anywhere?

Reply:
`cur`, `ico`, `svg`, `webp`